contracts/CollateralLockerFactory.sol

```diff
@@ -13,15 +13,20 @@ contract CollateralLockerFactory {
 
     uint8 public constant factoryType = 0;  // i.e FactoryType::COLLATERAL_LOCKER_FACTORY
 
+    event CollateralLockerCreated(address owner, address collateralLocker, address collateralAsset);
```

Comment:
it is better to `indexed` the owner, So offchain team can filter on the basis of `owner`